README.md

```diff
@@ -52,7 +53,7 @@ package in the image.
   - Nodes may have arbitrary hostnames but these should be lowercase to avoid a mismatch between inventory and actual hostname.
   - Nodes in a group are assumed to be homogenous in terms of processor and memory.
   - An inventory group may be empty, but if it is not then the play must contain at least one node from it (used to set processor information).
-
+* `ram_multiplier`: Optional, default `openhpc_ram_multiplier`. Allows for per group override of `openhpc_ram_multiplier`. Has no effect if `ram_mb` is set.
```

Comment:
As per e.g. cluster-name would it be better to describe this as "An override for the top-level definition" - so you don't have to mention "group" - b/c it could be partition-specific?